### ISSUE TITLE
join call without unlock screen

### ISSUE THREAD

**Issue by mahibi** [nextcloud/talk-android#1667]
## Steps to reproduce
1. Lock phone with lockscreen
2. call phone from web
3. (phone should wake up and show notification screen), accept call

### Expected behaviour
go to call without the need to unlock the phone after accepting.
Lockscreen should only be shown when user navigates to other screens (for privacy security reasons; also when entering picture-in-picture mode). 

### Actual behaviour
lockscreen is shown. Not able to join call without to unlock.

this behaviour occured when implementing https://github.com/nextcloud/talk-android/pull/1655
see comment https://github.com/nextcloud/talk-android/pull/1655/files#r738453308
see commit c0cca130ee84628510f7bc4e64ad3f93f8ab9d5f

**Comment by jookk** [comment]
Issue still persist on android 14. When I look at Applications - Special settings permissions, Talk app doesn't ask for Showing notifications on full screen and Turn on screen  
Receiving the call behaves badly for user to pick call. First unlock then picking call.

**Comment by mahibi** [comment]
> Talk app doesn't ask for Showing notifications on full screen and Turn on screen

But you can turn on these permissions, right?
I think the points mentioned in https://github.com/nextcloud/talk-android/issues/3957#issuecomment-2713481693 should be implemented to solve it

**Comment by jookk** [comment]
No, i cant enable show on the top permission in Special permission. Toogle isn't there for talk app

**Comment by mahibi** [comment]
yes, i think "show on the top permission in Special permission" won't be there as it's request is not implemented.
Showing notifications on full screen and Turn on screen could be enabled i think, but it's not the solution i guess.

**Comment by jookk** [comment]
I don't have Talk in Turning on screen permissions.
Yes, it's confusing for people to pickup the incoming call. Thanks
